frontend/pages/beta/model/[modelId]/access-request/new.tsx

```diff
@@ -120,6 +120,7 @@ export default function NewAccessRequest() {
                   variant='contained'
                   onClick={onSubmit}
                   loading={submitButtonLoading}
+                  data-test='submitAccessRequest'
```

Comment:
Picky, but I think `data-test` tags should explain what the element is. So this would be something like `data-test='submitAccessRequestButton'`. I find it helps massively with test readability